src/vmm/src/default_syscalls/filters.rs

```diff
@@ -96,6 +96,16 @@ pub fn default_filter() -> Result<SeccompFilter, Error> {
             allow_syscall(libc::SYS_mremap),
             // Used for freeing memory
             allow_syscall(libc::SYS_munmap),
+            // Used for reading the timezone in LocalTime::now()
+            allow_syscall_if(
```

Comment:
Is `Localtime::now`'s behavior the same on all toolchains?

Reply:
What about `aarch64`?

Reply:
yes. the `mmap` is triggered for both `x86_64` and `aarch64`